ui/components/src/ui-components/timeline/TimelineComponent.tsx

```diff
@@ -51,7 +53,7 @@ interface TimelineComponentState {
 }
 
 /** Component used to playback timeline data
- * @alpha
+ * @public

```

Comment:
@NancyMcCallB - This component is not ready to be made public as it currently exist. There is still prototype code that was never fully implemented. For example only "mini" mode is really the only mode supported. Milestones should be removed as those were only shown when not in "mini" mode. The wider mode was never vetted and no UX design was ever finalized. The wide mode option and milestones should be removed before making this public.